src/DotLiquid/Context.cs

```diff
@@ -654,39 +655,26 @@ private static object Liquidize(object obj)
             {
                 return liquidizableObj.ToLiquid();
             }
-            if (obj is string)
+            if (obj is string || obj is IEnumerable || obj is decimal || obj is DateTime || obj is DateTimeOffset || obj is TimeSpan || obj is Guid)
             {
                 return obj;
             }
             if (obj is IEnumerable)

```

Comment:
Remove this now, right?

Reply:
No, we just combined a bunch of lines into one. But you are right `obj is IEnumerable` is duplicated.